cli/context.go

```diff
@@ -253,6 +253,37 @@ func (c FFSContext) truncTableData(str string, collen int) string {
 	}
 }
 
+func (c FFSContext) PrintPrimaryOutputTSV(data [][]string, columnFilter []int) {
+        if c.Opt.Quiet {
+                return
+        }
+
+        if len(data) == 0 {
+                return
+        }
+
+        lens := make([]int, len(data[0]))
+        for _, row := range data {
+                for i, cell := range row {
+                        lens[i] = mathext.Max(lens[i], len([]rune(cell)))
+                }
+        }
+
+        for rowidx := range data {
+
+                {
+                        rowstr := ""
```

Comment:
The only thing I'm kinda worried here is escaping the values. You are currently simply concatenating them, but imo some values can contains tabs and/or linebreaks (at least the bookmark title).

The correct way should be to use the CSVWriter from "encoding/csv" - but if you say you are not familiar enough n go we can probably just merge it like this and I'll change i later to "encoding/csv".